solr/solr-ref-guide/src/major-changes-in-solr-9.adoc

```diff
@@ -80,6 +80,8 @@ All the usages are replaced by BaseHttpSolrClient.RemoteSolrException and BaseHt
 
 * SOLR-15409: Zookeeper client libraries upgraded to 3.7.0, which may not be compatible with your existing server installations
 
+* SOLR-11623: Every request handler in Solr now implements PermissionNameProvider. Any custom or 3rd party request handler must also do this
```

Comment:
Perhaps here and/or in upgrade-notes we should say that users may need to adjust their roles and permission mappings to accommodate these changes.

Reply:
+1.  This should be implied, as users shouldn't be dragging any config across major versions, but many will regardless so it's worth calling out IMO.